### ISSUE TITLE
Moments localization not working

### ISSUE THREAD

**Issue by glebkarandin** [arfedulov/semantic-ui-calendar-react#75]
I`m trying

```javascript
import * as moment from 'moment';
moment.locale('ru')
...
import { DateInput } from "semantic-ui-calendar-react";
...
```
or
```javascript
import * as moment from 'moment';
...
import { DateInput } from "semantic-ui-calendar-react";
...
class MyClass extends React.Component {
  constructor() {
    moment.locale('ru')
  }
}
```
Result in MyClass:
`console.log(moment.locale()) // ru`
in DateInput
`console.log(moment.locale()) // en`

I suggest adding a parameter "locale" and code for example
```javascript
<DateInput locale="ru" />
```
and in DateInput (BaseInput?)
```javascript
class DateInput {
  constructor(props) {
    if (props.locale) moment.locale(props.locale);
  }
}
```

**Comment by arfedulov** [comment]
It's strange but I don't have problems with setting locale.

Yeah, probably it is a good idea adding ``locale`` prop.

**Comment by glebkarandin** [comment]
In our project, localization work if source of Calendar is a part of project (as Component).

![calendar_as_a_part_of_project](https://user-images.githubusercontent.com/4863126/51588862-62ce8300-1ef6-11e9-96fa-72166cca184c.PNG)

In this case it`s Ok...

If you need help, i can add the prop.

**Comment by arfedulov** [comment]
Help always appreciated :smile:

**Comment by repaly** [comment]
#85

**Comment by arfedulov** [comment]
``localization`` prop added #85

**Comment by repaly** [comment]
Also this can be fixed by updating moment package to 2.24.0
